varnish/datadog_checks/varnish/varnish.py

```diff
@@ -44,6 +45,45 @@ class Varnish(AgentCheck):
     # Output of varnishstat -V : `varnishstat (varnish-4.1.1 revision 66bb824)`
     version_pattern = re.compile(r'(\d+\.\d+\.\d+)')
 
+    def __init__(self, name, init_config, instances):
+        super(Varnish, self).__init__(name, init_config, instances)
+        self.custom_tags = list(set(self.instance.get('tags', [])))
+
+        # Split the varnishstat command so that additional arguments can be passed in
+        # In order to support monitoring a Varnish instance which is running as a Docker
+        # container we need to wrap commands (varnishstat, varnishadm) with scripts which
+        # perform a docker exec on the running container. This works fine when running a
+        # single container on the host but breaks down when attempting to use the auto
+        # discovery feature. This change allows for passing in additional parameters to
+        # the script (i.e. %%host%%) so that the command is properly formatted and the
+        # desired container is queried.
+        self.varnishstat_path = self.instance.get('varnishstat', '').split()
+
+        self.varnishadm = self.instance.get("varnishadm", None)
+        # Split the varnishadm command so that additional arguments can be passed in
+        # In order to support monitoring a Varnish instance which is running as a Docker
+        # container we need to wrap commands (varnishstat, varnishadm) with scripts which
+        # perform a docker exec on the running container. This works fine when running a
+        # single container on the host but breaks down when attempting to use the auto
+        # discovery feature. This change allows for passing in additional parameters to
+        # the script (i.e. %%host%%) so that the command is properly formatted and the
+        # desired container is queried.
+        self.varnishadm_path = self.instance.get('varnishadm', '').split()
```

Comment:
Can we delete the code in `check` as well? https://github.com/DataDog/integrations-core/blob/master/varnish/datadog_checks/varnish/varnish.py#L138